lib/appbundler/cli.rb

```diff
@@ -9,10 +9,12 @@ class CLI
     banner(<<-BANNER)
 * appbundler #{VERSION} *
 
-Usage: appbundler APPLICATION_DIR BINSTUB_DIR
+Usage: appbundler BUNDLE_DIR BINSTUB_DIR [GEM_NAME] [GEM_NAME] ...
```

Comment:
I believe appbundle-updater will do the same thing as before ... we copy the Gemfile.lock to the target dir, so the only difference between our build now and our build before is that the lock will have more stuff in it than strictly necessary for the one project--but appbundler handles that just fine.